.github/workflows/ci.yaml

```
@@ -7,7 +7,7 @@ env:
 
 jobs:
   cabal-test:
-    name: cabal test
+    name: cabal test with GHC 9.4.1
```

Ah, this is causing the PR to be blocked. Change to
```suggestion
    name: cabal test
```